perma_web/tasks/dev.py

```diff
@@ -701,3 +707,496 @@ def populate_folder_cached_path(ctx, batch_size=500):
+        org_links = Link.objects.filter(created_by__in=to_delete, organization__isnull=False)
+        with open(TRANSFERRED_ORG_LINKS_CSV, 'a', newline='') as csvfile:
+            writer = csv.writer(csvfile, delimiter='|')
+            for link in org_links:
+                writer.writerow([link.guid, link.created_by_id, to_keep.id, timezone.now()])
+                updated_org_links[link.created_by_id] += 1
+        org_links.update(
+            created_by=to_keep
+        )
+
+        # Then, move all the Personal Links into the target account's Personal Links folder,
+        # in addition to setting 'created_by' to the new ID. We know from studying the accounts
+        # in question that folder tree structure can be ignored.
+        for user in to_delete:
+            personal_links = Link.folders.through.objects.filter(link__in=user.created_links.all())
+            with open(TRANSFERRED_PERSONAL_LINKS_CSV, 'a', newline='') as csvfile:
+                writer = csv.writer(csvfile, delimiter='|')
+                for lf in personal_links:
+                    writer.writerow([lf.link_id, lf.link.created_by_id, lf.folder_id, to_keep.id, to_keep.root_folder_id, timezone.now()])
+            updated_personal_links[user.id] += personal_links.update(folder_id=to_keep.root_folder_id)
+            user.created_links.all().update(created_by_id=to_keep.id)
+
+    with open(MERGED_USERS_CSV, 'a', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter='|')
+        for user in to_delete:
+            original_email, placeholder_email = user.soft_delete_after_merge_with_user(to_keep)
+            writer.writerow([
+                user.id,
+                original_email,
+                placeholder_email,
+                original_email.lower(),
+                to_keep.id,
+                updated_org_links[user.id],
+                updated_personal_links[user.id],
+                timezone.now()
+            ])
+
+    merged_with = ', '.join([str(user.id) for user in to_delete])
+    to_keep.prepend_to_notes(f"Merged with {merged_with}")
+    if updated_org_links or updated_personal_links:
+        to_keep.link_count = to_keep.created_links.count()
+    to_keep.save(update_fields=['notes', 'link_count'])
+    with open(RETAINED_USERS_CSV, 'a', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter='|')
+        writer.writerow([
+            to_keep.id,
+            to_keep.email,
+            to_keep.email.lower(),
+            merged_with,
+            timezone.now()
+        ])
+
+
+def unmerge_accounts(from_user_id):
+    from perma.models import Link, LinkUser
+
+    user = LinkUser.objects.get(id=from_user_id)
+    if match := re.search(r"\n*Merged with (?P<user_ids>.+)", user.notes):
+        user_ids = [int(uid) for uid in match.group('user_ids').split(', ')]
+    else:
+        print(f"Found no accounts to unmerge from user {from_user_id}.")
```

Comment:
Let's check where these prints will go during the prod run, and see if they would be better written to another report, or written to the application log.